frontend/src/components/visual-editor/form/PluginMessageForm.tsx

```
@@ -30,7 +30,7 @@ const PluginMessageForm = () => {
     {
       params: {
         plugin:
-          block?.message && "plugin" in block?.message
+          block.message && "plugin" in block?.message
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)